account_due_list_payment_mode/views/payment_view.xml

```diff
@@ -13,15 +12,14 @@
     </record>
 
     <record id="view_payments_mode_filter" model="ir.ui.view">
-        <field name="name">Payments Mode Select</field>
         <field name="model">account.move.line</field>
         <field name="inherit_id" ref="account_due_list.view_payments_filter"/>
         <field name="arch" type="xml">
             <field name="date_maturity" position="after">
                 <field name="payment_mode_id"/>
             </field>
-            <filter string="Month" icon="terp-go-month" domain="[]"
-                    context="{'group_by':'date_maturity'}" position="after">
+            <filter string="Month" icon="terp-go-month"
```

Comment:
You haven't understood me here. I want to not put the full line as selector, but only:
```
<filter context="{'group_by':'date_maturity'}" position="after">
```
I change it anyway